app/mailers/publishers/job_application_mailer.rb

```diff
@@ -0,0 +1,15 @@
+class Publishers::JobApplicationMailer < Publishers::BaseMailer
+  helper VacanciesHelper
+
+  def applications_received(publisher:)
+    @template = NOTIFY_PUBLISHER_JOB_APPLICATIONS_RECEIVED_TEMPLATE
+    @publisher = publisher
+    @vacancies = publisher.vacancies_with_job_applications_submitted_yesterday
+    @to = publisher.email
+
+    @job_applications_count = @vacancies.sum { |vacancy| vacancy.job_applications.count }
```

Comment:
Perhaps we want the vacancy id and the job applications data to be sent in the `email_event_data`?

Is it worth adding something for now, and then checking with @stevenleggdfe what he might need when he's back from leave?